### ISSUE TITLE
evapi: add module parameter to enable keepalive

### ISSUE THREAD

**Issue by linuxmaniac** [kamailio/kamailio#1290]
I'm having issues when I reach EVAPI_MAX_CLIENTS connections but some of them are just lingering and they are never release, so no new connections can be handled.

Maybe keepalive will help detecting those lingering connections and release them

**Comment by miconda** [comment]
I went and added this in order to allow coping in such situations, being a limitation in some operation cases, it needed to be fixed somehow.

On the other hand, while doing it, I noticed that a close was missing, so it could have been a reason for some connections to stay longer in open state, but it should have not affected the number of client structure, only file descriptors.

I also enabled tcp keepalive on client connections with another commit.

Testing, especially if max_clients doesn't break things, would be appreciated.
